at-chuongphu/excercise5.rb

```diff
@@ -0,0 +1,41 @@
+# ex1
+class Student
+	attr_accessor :first_name, :last_name, :list_course
+  def initialize
+    @list_course = []
+  end
+  def first_name
+    puts "#{@first_name}"
+  end
+  def last_name
+    puts "#{@last_name}"
+  end
+  def full_name
+    puts "#{@first_name}" + " #{@last_name}"
+  end
+  def add_course(course_name)
+    # @list_course = []
+    @list_course.push(course_name) unless @list_course.include?(course_name)
+	end
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/1c0a5adcb251976aac9f1128795ec3dbcba91c88/at-chuongphu/excercise5.rb#L19"></span>Layout/DefEndAlignment: <code>end</code> at 19, 1 is not aligned with <code>def</code> at 16, 2.</td>
    </tr>
  </tbody>
</table>